### PR TITLE
Update export URL according to README - Partially Closes 1838

### LINKED ISSUE #1838: Update overview page of Service docs
### Description

Update the Lisk Service docs according to latest README changes.

### Additional Information
https://github.com/LiskHQ/lisk-service/pull/1836/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R44

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Partially closes https://github.com/LiskHQ/lisk-service/issues/1838

### CODE REVIEW

File: docs/modules/ROOT/pages/api/lisk-service-rpc.adoc [hunk 1]
```diff
@@ -7899,7 +7899,7 @@ After a few seconds, re-request the same API with the same credentials to get a
 
 Upon receiving a similar response as mentioned above, combine the `fileUrl` with the HTTP base URL: https://service.lisk.com like the following to download the file.
```
Comment (chris529) [inline docs/modules/ROOT/pages/api/lisk-service-rpc.adoc]:
```suggestion
Upon receiving a similar response as mentioned above, combine the `fileUrl` with the HTTP base URL: https://service.lisk.com to download the file as shown in the following below:
```

Comment (TalhaMaliktz) [reply docs/modules/ROOT/pages/api/lisk-service-rpc.adoc]:
```suggestion
Upon receiving a similar response as mentioned above, combine the `fileUrl` with the HTTP base URL: https://service.lisk.com to download the file as shown in the following URL:
```

**Review by Tschakki — APPROVED** [review]
LGTM

**Review by chris529 — APPROVED** [review]
LGTM! I just left a small suggestion 👍

**Pull request merged**